src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
```

Comment:
The class `Circle` should not start with an empty line. Please remove the empty line to follow good coding practices.